.github/workflows/sync-kubernetes-configuration-crds.yaml

```diff
@@ -8,6 +8,10 @@ on:
         required: true
         type: string
 
+permissions:
+  contents: write
+  pull-requests: write
```

Comment:
Given we're relying on `K8S_TEAM_BOT_GH_PAT` secret, I think these permissions are not necessary.

Reply:
Good catch! I forgot to update it. Thank you 👍